lib/srv/sshserver.go

```diff
@@ -692,6 +689,10 @@ func (s *Server) handleSessionRequests(sconn *ssh.ServerConn, ch ssh.Channel, in
 			}
 		case result := <-ctx.result:
 			ctx.Infof("ctx.result = %v", result)
+			// pass back stderr output
+			if result.stderr != nil {
```

Comment:
probably more robust way to check to avoid zero writes would be `if len(result.stderr) != 0`